doc/suda.txt

```diff
@@ -40,10 +40,21 @@ Make sure that the following shows 1.
 =============================================================================
 USAGE						*suda-usage*
 
-Add "suda://" prefix (see |g:suda#prefix|) to the filename and use Vim's
-builtin |read|, |edit|, |write|, |saveas| commands.
+Use |SudaRead| and |SudaWrite| commands like:
```

Comment:
Reference does not exist for `:SudaRead` and `:SudaWrite`.
(And leading `:` is missing.)